soccer/gameplay/robocup-py.cpp

```diff
@@ -94,6 +95,11 @@ void OurRobot_move_to_direct(OurRobot* self, Geometry2d::Point* to) {
     if (to == nullptr) throw NullArgumentException("to");
     self->moveDirect(*to);
 }
+void OurRobot_move_to_direct_end_vel(OurRobot* self, Geometry2d::Point* to,
```

Comment:
It may be best if we make "ball placement" its own game state alongside stopped and restart, since it's not really either one of those.  Then we can just not put an obstacle around the ball when it's our placement and you can use the normal move() method.